engine/src/asset/texture/dat/offset.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{
+    asset::{extension::*, AssetParser},
+    utils::nom::*,
+};
+
+pub struct TextureOffset {
```

Comment:
This is the `TextureInfo` file right? Shouldn't we call it the same way? (I guess we already have the "convention" to call these "Offsets".

Reply:
Even if the asset is called `TEXTURE_INFO`, according to debug symbols, the class itself is called `TextureOffsets`.

![image](https://github.com/nenikitov/ashen-decomp/assets/44075969/d9ab97e1-e227-419a-a4a6-4f0ca0091289)

I prefer offsets, it seems more descriptive to me.